docs/docs/running-node/auth.md

```diff
@@ -0,0 +1,49 @@
+# Authentication and authorization
+
+Bacalhau includes a flexible auth system that supports multiple methods of auth
+that are appropriate for different deployment environments.
+
+## By default
+
+With no specific authentication configuration supplied, Bacalhau runs in
+"anonymous mode" – which allows unidentified users limited control over the
+system. "Anonymous mode" is only appropriate for testing or evaluation setups.
+
+In anonymous mode, Bacalhau will allow:
+
+- Users identified by a self-generated private key to submit any job, cancel
+  their own jobs, read job lists and use `bacalhau describe`.
```

Comment:
Should this be `bacalhau job describe` instead as we are moving away from the latter?

Reply:
I will just change it to not mention the explicit command.